scheduler/kubernetes/csi.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: page
+title: "Deploying Portworx with CSI support"
+keywords: csi, portworx, container, Kubernetes, storage, Docker, k8s, pv, persistent disk 
+sidebar: home_sidebar
+---
+
+* TOC
+{:toc}
+
+[CSI](https://kubernetes-csi.github.io/), or _Container Storage Interface_, is
+the new model for integrating storage system service with Kubernetes and other
+orchestration systems. Kubernetes has had support for CSI since 1.10.
+
+With CSI, Kubernetes gives storage drivers the opportunity to release on their
+schedule. This allows storage vendors to upgrade, update, and enhance their drivers
+without the need to update Kubernetes, maintaining a consistent, dependable,
+orchestration system.
+
+## Install
+
+### Install using the Portworx spec generator
+When installing Portworx through [install.portworx.com](https://install.portworx.com)
+you can select CSI as the model to use for deployment.
+
+If you are using [curl to fetch the Portworx
+spec](https://docs.portworx.com/scheduler/kubernetes/px-k8s-spec-curl.html), you can add
+`csi=true` to the parameter list to include Stork specs in the generated file.
+
+## Impact on applications
+
+The only affected object is the [_StorageClass_](https://kubernetes-csi.github.io/docs/Usage.html#dynamic-provisioning).
+For any StorageClasses created, you will need to setup the value of `provisioner`
+to `com.openstorage.pxd`. Here is an example:
+
+```yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: portworx
+provisioner: com.openstorage.pxd
+  repl: "3"
```

Comment:
indentation for provisioner
and parameters section

Reply:
provisioner indentation is correct, but you I missed the parameters section.